modules/camera/camera_opencv.py

```diff
@@ -63,4 +63,4 @@ def run(self) -> tuple[True, np.ndarray] | tuple[False, None]:
         if not result:
             return False, None
 
-        return True, image_data
+        return True, image_data
```

Comment:
Add newline at end of file back. They usually mark the end of file, so don't delete them